Task4/index.html

```diff
@@ -0,0 +1,41 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+	<link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="script.js"></script>
+</head>
+<body>
+    <div id="phone">
+        <div id="screenContainer">
+		<img id="screen" src="Images/wallpaper.jpg" alt="Wallpaper of snowy mountaint at night">  
```

Comment:
The same with tabs here.

```suggestion
            <img id="screen" src="Images/wallpaper.jpg" alt="Wallpaper of snowy mountaint at night">  
```